src/main/java/service/UserService.java

```diff
@@ -0,0 +1,36 @@
+package service;
+
+import entity.User;
+import repository.UserRepository;
+
+import javax.servlet.http.HttpServletRequest;
+
+public class UserService {
+    private static final UserService instance = new UserService();
```

Comment:
If you trying to create singleton (I think you are trying because of getInstance() method) you should to define private constructor (that nobody will be able to create instance of UserService with ordinal new UserService() statement;)